### PR TITLE
Add first deployment

### LINKED ISSUE #10: Deployment setup using Github actions
Find a way to deploy the website on a staging environment so we can share with Macou.

### PR DESCRIPTION
**Pull request by GabCh**
https://gabch.github.io/nodes-website/

closes #10

### CODE REVIEW

File: .github/workflows/CI-workflow.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: CI
 # This workflow is triggered on pushes to the repository.
```
Comment (Gab05) [inline .github/workflows/CI-workflow.yml]: update comment

Comment (GabCh) [reply .github/workflows/CI-workflow.yml]: Indeed, I updated it!

**Comment by GabCh** [comment]
I forgot to remove the build folder! I just did now.

**Comment by GabCh** [comment]
I'll wait a little bit before the merge. I want Macou to revise the new design I manually deployed last night before overriding it with the current one.

**Pull request merged**